symengine/printers/strprinter.cpp

```diff
@@ -1020,6 +1020,16 @@ void StrPrinter::bvisit(const Tuple &x)
     str_ = o.str();
 }
 
+void StrPrinter::bvisit(const IdentityMatrix &x)
+{
+    str_ = "I";
```

Comment:
This is the same as the complex number sqrt(-1). Maybe we should have a different string. What does sympy do?

Reply:
sympy uses `I` for both. Parses to imaginary unit